cluster/config.go

```diff
@@ -111,6 +124,11 @@ func BuildContainerConfig(c dockerclient.ContainerConfig) *ContainerConfig {
 		}
 	}
 
+	// store reschedule policy in labels
+	if reschedulePolicy != "" {
+		c.Labels[SwarmLabelNamespace+".reschedule-policy"] = reschedulePolicy
```

Comment:
If there is only a very limited number of reschedulePolicy, I think we should fail it explicitly. For example, user may input `-e reschedule:On-node-failure` with a capital `O`, or `-e reschedule:node-failure`. It's better to let user know it's wrong, than failing it later implicitly.